examples/step-74/doc/results.dox

```diff
@@ -0,0 +1,195 @@
+<h1>Results</h1>
+
+The output of this program consist of the console output and
+solutions in vtu format.
+
+Convergence rate for the smooth case with polynomial degree 3:
+@code
+DEAL::Cycle 0
+DEAL::Number of active cells:       16
+DEAL::Number of degrees of freedom: 256
+   Solving system...
+DEAL::Writing solution to <sol_Q3-00.vtu>
+   Error in the L2 norm       :     0.00193285
+   Error in the H1 seminorm   :     0.106087
+   Error in the enery norm    :     0.150625
+
+DEAL::Cycle 1
+DEAL::Number of active cells:       64
+DEAL::Number of degrees of freedom: 1024
+   Solving system...
+DEAL::Writing solution to <sol_Q3-01.vtu>
+   Error in the L2 norm       :     9.60497e-05
+   Error in the H1 seminorm   :     0.0089954
+   Error in the enery norm    :     0.0113265
+
+DEAL::Cycle 2
+DEAL::Number of active cells:       256
+DEAL::Number of degrees of freedom: 4096
+   Solving system...
+DEAL::Writing solution to <sol_Q3-02.vtu>
+   Error in the L2 norm       :     5.60638e-06
+   Error in the H1 seminorm   :     0.000901791
+   Error in the enery norm    :     0.000973568
+
+DEAL::Cycle 3
+DEAL::Number of active cells:       1024
+DEAL::Number of degrees of freedom: 16384
+   Solving system...
+DEAL::Writing solution to <sol_Q3-03.vtu>
+   Error in the L2 norm       :     3.48399e-07
+   Error in the H1 seminorm   :     0.000107055
+   Error in the enery norm    :     0.000108757
+
+DEAL::Cycle 4
+DEAL::Number of active cells:       4096
+DEAL::Number of degrees of freedom: 65536
+   Solving system...
+DEAL::Writing solution to <sol_Q3-04.vtu>
+   Error in the L2 norm       :     2.17928e-08
+   Error in the H1 seminorm   :     1.32657e-05
+   Error in the enery norm    :     1.33065e-05
+
+DEAL::Cycle 5
+DEAL::Number of active cells:       16384
+DEAL::Number of degrees of freedom: 262144
+   Solving system...
+DEAL::Writing solution to <sol_Q3-05.vtu>
+   Error in the L2 norm       :     1.36274e-09
+   Error in the H1 seminorm   :     1.65576e-06
+   Error in the enery norm    :     1.65683e-06
+
+degree = 3
+| cycle | cells | dofs   | L2        | L2...red.rate.log2 | H1        | H1...red.rate.log2 | Energy    |
```

Comment:
I think it would be easier to read if you just had
```suggestion
| cycle | cells | dofs   | L2        | L2 reduction factor | H1        | H1 reduction factor | Energy    |
```